blog/02-20-2023-how-to-assemble-a-web3-dream-team-and-build-a-high-value-dapp/how-to-assemble-a-web3-dream-team-and-build-a-high-value-dapp.md

```diff
@@ -0,0 +1,140 @@
+---
+title: How to Assemble a Web3 Dream Team and Build a High Value dApp
+description: There are multiple factors to consider when assembling a web3 dream team.
+
+authors:
+  - name: Alice Huang
+    title: Management Consultant
+    url: https://github.com/LilMissGiggles
+tags: [celo, dapp, beginner, celosage]
+hide_table_of_contents: true
+slug: /tutorials/how-to-assemble-a-web3-dream-team-and-build-a-high-value-dapp
+---
+
+# How to Assemble a Web3 Dream Team and Build a High Value dApp
```

Comment:
We are supposed to header Image, and remove the title as It will be added automatically from the metadata

Reply:
You can get the header image from the trello card , download the image from that and save that image in 
src/data-tutorials/showcase/beginner/NAME-OF-THE-IMAGE.extension) and you can add the in your .md with adding such link

![header](../../src/data-tutorials/showcase/beginner/NAME_OF_IMAGE.extension)

Reply:
this was the line 
" ![header](../../src/data-tutorials/showcase/beginner/build-in-public.png) "

Reply:
'![header](../../src/data-tutorials/showcase/beginner/build-in-public.png)'

Reply:
I have written the link on your trello card